torch/utils/cpp_extension.py

```diff
@@ -271,34 +290,35 @@ def check_compiler_ok_for_platform(compiler: str) -> bool:
     return False
 
 
-def check_compiler_abi_compatibility(compiler) -> bool:
+def get_compiler_abi_compatibility_and_version(compiler) -> Tuple[bool, Tuple[str, ...]]:
```

Comment:
If compiler version is `minor.major`, then why not return `float` or `Tuple[int, int]`, rather than obscure `Tuple[str, ...]`?
If it's a [semver](https://semver.org/), than please use `Tuple[int, int, int]`